x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/profiler/ilm-policy/profiling.json

```diff
@@ -1,12 +1,10 @@
 {
   "phases": {
     "hot": {
-      "min_age": "0ms",
       "actions": {
         "rollover": {
           "max_primary_shard_size": "50gb",
-          "max_age": "7d",
-          "min_docs": 1
+          "max_age": "15d"
```

Comment:
`min_docs` is not needed because ILM would not rollover empty indices by default anyway. We half one rollover after half the hot retention period to avoid the risk of querying the warm tier too often.